qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -11,6 +11,8 @@
     gagent_pkg_check_cmd = "rpm -q qemu-guest-agent"
     black_list_change_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"
     black_list_check_cmd = "grep -nr 'BLACKLIST_RPC=.*%s' /etc/sysconfig/qemu-ga"
+    black_list_backup = "/bin/cp -rf /etc/sysconfig/qemu-ga /etc/sysconfig/qemu-ga-bk"
```

Comment:
Why `-r`?

Reply:
just regard any file as normal file to copy,used to use it .if there is no need to add it ,I'll delete it .

Reply:
`-r` is for directory but here is a file so not necessary.